demo/components/group-demo.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
```

Comment:
The "shape" of this demo component is
```
<div>
  <VictoryChart>
    <VictoryStack>
      <VictoryGroup>
        <VictoryBar/>
        <VictoryLine/>
      </VictoryGroup>
      <VictoryGroup>
        <VictoryBar/>
        <VictoryLine/>
      </VictoryGroup>
      <VictoryGroup>
        <VictoryBar/>
        <VictoryLine/>
        <VictoryScatter/>
      </VictoryGroup>
    </VictoryStack>
  </VictoryChart>

  <VictoryChart>
    <VictoryGroup>
      <VictoryLine/>
      <VictoryBar/>
      <VictoryErrorBar/>
    </VictoryGroup>
  </VictoryChart>
</div>
```

This might be a little complex to convey the concept. There are two `VictoryChart`s "grouped" in a div, and then each has a `VictoryGroup` grouping further.

I also mentioned below that the extra layer of the `VictoryStack` might be a little complex.

But I haven't run this locally, so I don't know how awesome it looks :)

Maybe this would make a good recipe?? (In the current "long" form)